.circleci/config.yml

```diff
@@ -1,7 +1,8 @@
 version: 2.1
 jobs:
   test-backend:
-    machine: true
+    machine:
```

Comment:
Noting that these CircleCI config changes are incidental to this PR and were already merged in via #69.